config-windows.md

```diff
@@ -112,11 +112,30 @@ For more information about tooling to generate a gMSA, see [Deployment Overview]
 When a container terminates, the Host Compute Service indicates if a Windows update servicing operation is pending.
 You can indicate that a container should be started in a mode to apply pending servicing operations via the OPTIONAL `servicing` field of the Windows configuration.
 
-
 ### Example
 
 ```json
     "windows": {
         "servicing": true
     }
+```
+
+## <a name="configWindowsHyperV" />HyperV
+
+`hyperv` is an OPTIONAL field of the Windows configuration. If present, the container MUST be run with Hyper-V isolation. If omitted, the container MUST be run as a Windows Server container.
+
+The following parameters can be specified:
+
+* **`utilityvmpath`** *(string, OPTIONAL)* - specifies the path to the image used for the utility VM. If not supplied, the runtime will search the container filesystem layers from the bottom-most layer upwards, until it locates "UtilityVM", and default to that path.
```

Comment:
How about I update the comment to

`specified the path to the image used for the utility VM. This would be specified if using a base image which does not contain a utility VM image. If not supplied, the runtime will....` and add the example then? To be fair `\\path\\to\\utilityvm` is probably about as meaningful an example as I could get regardless 😸